### ISSUE TITLE
ApplicationRecord subclass cop runs on ApplicationRecord model

### ISSUE THREAD

**Issue by chrisjoywebb** [github/rubocop-github#2]
Thanks for putting this together! It's a great way to stay current with GitHub's style recommendations.

One small issue that I've noticed is that the ApplicationRecord subclass cop runs on `application_record.rb` in addition to other models. This is what I'm getting:

```
app/models/application_record.rb:2:27: C: Models should subclass from ApplicationRecord
class ApplicationRecord < ActiveRecord::Base
                          ^^^^^^^^^^^^^^^^^^
```

Changing this results in a circular dependency issue, of course. Just wanted to point it out. 😄

**Comment by chrisjoywebb** [comment]
Added a PR for this issue here: https://github.com/github/rubocop-github/pull/3, if it's at all helpful.

**Comment by josh** [comment]
Fixed by merging #3. Thanks!!!
